src/glob.rs

```diff
@@ -44,14 +46,32 @@ fn determine_type(glob: &str) -> GlobType {
     }
 }
 
-#[derive(Clone, PartialEq)]
+#[derive(Clone)]
 pub struct Glob {
     glob: GlobType,
     weight: i32,
     case_sensitive: bool,
     mime_type: Mime,
 }
 
+impl PartialEq for Glob {
+    fn eq(&self, other: &Glob) -> bool {
+        self.glob == other.glob &&
+        self.case_sensitive == other.case_sensitive &&
```

Comment:
Case sensitivity is also not taken into account in `xdgmime` when checking for insertion.

Reply:
Ok I'll fix that.